go.mod

```diff
@@ -1,61 +1,62 @@
 module github.com/IBM/ibm-storage-odf-block-driver
 
-go 1.18
+go 1.19
 
 require (
-	github.com/IBM/ibm-storage-odf-operator v1.4.0
-	github.com/prometheus/client_golang v1.13.0
+	github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230821022608-89c7031812eb
+	github.com/prometheus/client_golang v1.16.0
 	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
-	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v0.25.0
 	k8s.io/klog v1.0.0
-	sigs.k8s.io/controller-runtime v0.13.0
+	sigs.k8s.io/controller-runtime v0.12.3
```

Comment:
why is this older?

Reply:
because that's the version that fits to all the other packages we use. 
It has been updated automatically after we choose apimachinery 0.27 and client-go 0.25